Csv fix

josenino95 opened the pull request:
_If this pull request addresses an open issue on the repository, please add 'Closes #NN' below, where NN is the issue number._
Closes #89 and #114

_Please briefly summarise the changes made in the pull request, and the reason(s) for making these changes._
Modify episodes 5 and 6 to include a description of how to open a CSV file. This potentially resolves old issues #89 and #114, as the changes make more explicit how to open a csv in Excel or Calc.

I also added a second commit to delete images not needed in the fig/ folder, and to resize some png screenshots so episode 5 displays better when you switch between the Excel and Calc tab instructions.

_If any relevant discussions have taken place elsewhere, please provide links to these._
Relevant discussion in issue #114

<details>

For more guidance on how to contribute changes to a Carpentries project, please review [the Contributing Guide](CONTRIBUTING.md) and [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html).

Please keep in mind that lesson Maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

</details>

allie-tatarian reviewed (commented):
Thanks for putting this together Jose - I think it looks great! I added a few comments about some potential light changes. 

For the images - did you change these so that the height of the Calc screenshots matches the Excel screenshots, so the page jumps less when you switch between tabs? I'm of two minds about this - I understand why you want to do this, but I'm worried that the new images are a little blurry so they might be hard to see, and I don't know how much learners will realistically be switching between tabs, since they will likely just be using one program or the other. I am mostly worried from an accessibility standpoint - right now, the font size on the screenshots matches the font size of the lesson, but the resize will make it smaller. I feel like the page jumping issue will mostly affect instructors, so is there another solution that we could apply to only the instructor view? Or - could we keep the images as is but change their size on the page, so that the smaller images are higher resolution?

I do think that clearing out the extra images is fine, and will help streamline that folder and make it more usable. 

Thanks again for doing all of this!

josenino95 commented on episodes/04-quality-assurance.md:
```diff
@@ -42,8 +42,40 @@ We will look at two examples:
 
 ### Restricting data to a numeric range
 
-Looking again at the [clean version of the SAFI
```
Yes, I think it's worthy of an instructor note. I'll make a commit for this and addressing your other comments

josenino95 commented:
I made two commits addressing all your comments, @allie-tatarian. They were all super helpful, and I agree with everything. Hopefully the changes I made resolve them, but please add more comments if they don't.

allie-tatarian reviewed (approved):
This looks great - the new screenshots are helpful, thanks for adding those and the alt text. I'm excited for these changes to go live!
